### DISCUSSION TITLE
League/Commonmark

### DISCUSSION THREAD

**Discussion by ghost** [tighten/jigsaw#634] [Ideas]
Have you guys consider moving away from the current markdown and use League/CommonMark
https://commonmark.thephpleague.com/

I guess is more of an idea but it has a lot cool features.

**Comment by bakerkretzmar** [comment]
Yep, love CommonMark and we're going to be using it in v2 👍🏻

**Reply by ghost** [reply]
very nice!  how far is v2

**Reply by spekulatius** [reply]
Nice, I'm very happy to hear this :+1:

**Reply by walirazzaq** [reply]
CommonMark is gonna give superpowers to Jigsaw 🚀 
Can't wait for `v2`. Do we have any roadmap to follow?

**Comment by ghost** [comment]
how far is v2

**Comment by iSWORD** [comment]
This has been out for a while! 🚀 

https://github.com/tighten/jigsaw/pull/687
